### ISSUE TITLE
Meow version

### ISSUE THREAD

**Issue by GOMMB** [sass/node-sass#2905]
I'm not sure if this has been addressed yet but the version of meow that node-sass is using depends on an insecure version of minimist as stated [here](https://github.com/advisories/GHSA-vh95-rmgr-6w4m) and so meow should be updated to at least v5.0.0.

**Comment by nschonni** [comment]
We don't ship a package-lock.json or shrinkwrap file, so there is nothing to patch here.
EX: from installin master with a clean lock and node_modules:
```
    "meow": {
      "version": "3.7.0",
      "resolved": "https://registry.npmjs.org/meow/-/meow-3.7.0.tgz",
      "integrity": "sha1-cstmi0JSKCkKu/qFaJJYcwioAfs=",
      "requires": {
        "camelcase-keys": "^2.0.0",
        "decamelize": "^1.1.2",
        "loud-rejection": "^1.0.0",
        "map-obj": "^1.0.1",
        "minimist": "^1.1.3",
        "normalize-package-data": "^2.3.4",
        "object-assign": "^4.0.1",
        "read-pkg-up": "^1.0.1",
        "redent": "^1.0.0",
        "trim-newlines": "^1.0.0"
      }
```
The `^` will use the patched version of minimist 1.5+ unless there is something conflicting in your own lockfile. The only dependency currently pinned to an old minimist version is mocha, but that's a devDependency and won't affect your install

**Comment by GOMMB** [comment]
My apologies, I always get the ^ and ~ mixed up.

**Comment by nschonni** [comment]
No worries, the package-lock generation can also be a little hard to follow if it's not doing what you think it should
